script/script.js

```diff
@@ -191,56 +281,69 @@ const datatable = {
-                                    return value >= 2000 && value < 2010;
-                                case "new":
-                                    return value >= 2010;
+            on: {
+                onAfterLoad: function () {
+                    this.registerFilter(
+                        $$("tabbar"), {
+                            columnId: "year",
+                            compare: function (value, filter) {
+                                switch (filter) {
+                                    case "all":
+                                        return true;
+                                    case "old":
+                                        return value < 2000;
+                                    case "modern":
+                                        return value >= 2000 && value < 2010;
+                                    case "new":
+                                        return value >= 2010;
+                                }
+                            }
+                        }, {
+                            getValue: function (view) {
+                                return view.getValue();
+                            },
+                            setValue: function (view, value) {
+                                view.setValue(value);
                             }
                         }
-                    }, {
-                        getValue: function (view) {
-                            return view.getValue();
-                        },
-                        setValue: function (view, value) {
-                            view.setValue(value);
-                        }
-                    }
-                );
+                    );
+                }
             },
             onClick: {
                 "wxi-trash": function (e, id) {
-                    this.remove(id);
-                    const values = $$("form").getValues();
-                    if (id.row === values.id) {
-                        clearFields();
-                    }
-                    return false;
+                    webix.confirm({
+                        text: "Record will be deleted permanently! Continue?"
+                    }).then(() => {
+                        this.remove(id);
+                        const values = $$("form").getValues();
+                        if (id.row === values.id) {
+                            clearFields();
+                        }
+                        return false;
+                    });
                 }
             },
             scheme: {
                 $init: function (obj) {
-                    let votes = obj.votes;
+                    let votes = obj.votes,
+                        rating = obj.rating;
+
                     if (votes.includes(",")) {
-                        obj.votes = parseFloat(votes.replace(",", ".")) * 1000;
+                        obj.votes = (+(votes.replace(",", "."))) * 1000;
```

Comment:
done, but I multiplied by 1000 because I saw such values in the votes column on the screen in our lesson task

![image](https://user-images.githubusercontent.com/54096402/75956478-0eebe700-5ec9-11ea-9858-fcfbd2ae70cd.png)

Reply:
If you replace comma to dot like `(",", ".")`, you get a float number which is 1000 times smaller than the initial. And you have to multiply. 

But if you replace comma to an empty string `(",", "")`, you will get the correct number without the need to multiply.

Reply:
Oh, great, didn't notice